code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,6 +136,27 @@
 	taste_description = "rich earthy pungent"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/nutriment/clothiment
+	name = "Clothiment"
+	description = "It's not actually a form of nutriment but it does keep Moth people going for a short while..."
+	nutriment_factor = 30 * REAGENTS_METABOLISM
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	brute_heal = 0
+	burn_heal = 0
+	///Amount of satiety that will be drained when the clothiment is fully metabolized
+	var/delayed_satiety_drain = 30
+
+/datum/reagent/consumable/nutriment/clothiment/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(M.satiety < 600)
```

Comment:
What is this number?

Reply:
This is a copy from the base nutriment reagent. From what I understood it is a cap to how much nutriment(reagent) will be turned into nourishment(hunger) as it takes some time to metabolize, without it you would become fat a lot easier.

Reply:
I went to create a define for this value and we actually had one already, so I will change it on both to make it easier to read.